pkg/codegen/python/gen_test.go

```diff
@@ -51,7 +51,17 @@ func TestGeneratePackage(t *testing.T) {
 				assert.Contains(t, files, filepath.Join("pulumi_example", "other_resource.py"))
 
 				for fileName, file := range files {
+					if fileName == filepath.Join("pulumi_example", "resource.py") {
+						// Correct parent class
+						assert.Contains(t, string(file), "class Resource(pulumi.CustomResource):")
```

Comment:
If `isComponent` is true, shouldn't this be:

```suggestion
						assert.Contains(t, string(file), "class Resource(pulumi.ComponentResource):")
```

?

Reply:
`isComponent` is false for `resource.py` and true for `other_resource.py`, so this is correct.

Reply:
Ah, ok. I was looking at an earlier commit where they were both set to true in the schema.